src/Symfony/Component/Serializer/Tests/Context/Encoder/CsvEncoderContextBuilderTest.php

```diff
@@ -69,6 +69,19 @@ public static function withersDataProvider(): iterable
             CsvEncoder::OUTPUT_UTF8_BOM_KEY => false,
         ]];
 
+        yield 'With false key separator' => [[
```

Comment:
You can just add `CsvEncoder::KEY_SEPARATOR_KEY => false` in the previous yield instead of yielding a new set